EnglishFilter/sections/foreign.txt

```diff
@@ -788,6 +788,14 @@ kliuki.bg###container > div[style="float:left;"] > div[style]
 ! https://github.com/AdguardTeam/AdguardFilters/issues/26330
 ||acs.youku.com/*.youku.play.$replace=/("ad":{).+"(\}\,"playlog")/\$1\$2/
 @@||cibntv.net/youku/*.mp4?$domain=v.youku.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/32822
+zmz2019.com##.key-result > li > a[href*=".taobao.com"]
+||jstucdn.com/g3/
+.com/js/ad_$domain=zmz2019.com
+||nqtnu.cn/V.aspx?Bln=$domain=zmz2019.com
```

Comment:
EasyList China already his this rule, maybe apply it to our filter?

/V.aspx?Bln=$script,third-party


Or maybe we should fork EasyList China and make our own Chinese filter? Currently we have 732 rules for Chinese (comments counted.

Reply:
Some rules from EasyList China - they block ads good and on this site i didn't notice something broken.
For this PR I limited to domain cause idk should we add this rules to our filters (in future maybe our own Chinese):

`|http://ad.`
`.com/js/ad_`
`/V.aspx?Bln=$script,third-party`